Core/src/org/sleuthkit/autopsy/datamodel/VirtualDirectoryNode.java

```diff
@@ -80,13 +87,28 @@ public VirtualDirectoryNode(VirtualDirectory ld) {
      * @return
      */
     @Override
+    @NbBundle.Messages({"VirtualDirectoryNode.action.runIngestMods.text=Run Ingest Modules"})
     public Action[] getActions(boolean popup) {
         List<Action> actions = new ArrayList<>();
+
         actions.add(new NewWindowViewAction(
                 NbBundle.getMessage(this.getClass(), "VirtualDirectoryNode.getActions.viewInNewWin.text"), this));
         actions.add(null); // creates a menu separator
         actions.add(ExtractAction.getInstance());
         actions.add(null); // creates a menu separator
+        
+        if (this.content.isDataSource()) {
+            actions.add(new FileSearchAction(
```

Comment:
I have verified that we want the ability to run ingest modules on non-data source virtual directories. Please remove the isDataSource check.